src/cmd/flux-kvs.c

```diff
@@ -94,6 +94,9 @@ static struct optparse_option put_opts[] =  {
     { .name = "no-merge", .key = 'n', .has_arg = 0,
       .usage = "Set the NO_MERGE flag to ensure commit is standalone",
     },
+    { .name = "append", .key = 'A', .has_arg = 0,
```

Comment:
Any reason not to use little a for the short opt?

Reply:
little a is used for the "at" option throughout flux kvs, so picked big A just to be different.

Reply:
Oh duh.